test/src/e2e_vm_tests/test_programs/should_pass/language/panic_expression/panic_handling_in_unit_tests/snapshot.toml

```
@@ -0,0 +1,4 @@
+cmds = [
+    "forc test --path {root} --experimental error_type --logs --raw-logs --dbgs --reverts passing_",
```

nit: trailing underscore?
```suggestion
    "forc test --path {root} --experimental error_type --logs --raw-logs --dbgs --reverts passing",
```

---

it's a test filter, i assume the point is to find anything that matches "passing_*" and not "passing"

---

Exactly. The underscore is intentional.